Makefile

```diff
@@ -3,20 +3,21 @@ PACKAGES := $(notdir ${PACKAGE_DIRS})
 PACKAGE_TESTS := $(PACKAGES:%=test-%)
 R_VERSION := 3.5.3
 
+# Docker image is build on Docker Hub https://hub.docker.com/repository/docker/rstudio/shinyapps-package-dependencies/
```

Comment:
Docker image is built?

Reply:
I speak English good on Monday after vacation